Dima_Turchyn/src/HelloWorld.java

```diff
@@ -0,0 +1,6 @@
+public class HelloWorld {
+    public static void main(String[] args) {
+
```

Comment:
Don't leave empty line between opening/closing brases and code lines. It is not corresponding to Java codestyle
(I know I haven't mentioned it before but it would be nice if you make changes 😊)